### ISSUE TITLE
resolve-ref Reference does not exist for paths that contain ~1

### ISSUE THREAD

**Issue by stevenj** [Redocly/openapi-cli#124]
If i put ~1 in an external reference, i am getting resolve-ref errors.
The file does exist at the path, and other validators validate against this construct OK.

```openapi.yml:111:5 at #/paths/['/v1/adapter/claim']/$ref
Reference does not exist.

109| 
110|   /v1/adapter/claim:
111|     $ref: "v1/adapter/claim.oas3.yml#/paths/~1adapter~1claim"

Error was generated by resolve-ref rule.```

**Comment by adamaltman** [comment]
Just confirming, the path referenced in that remote file is `/adapter/claim`?

**Comment by stevenj** [comment]
yes it is

**Comment by RomanHotsiy** [comment]
@stevenj what version of openapi-cli you use? It should be fixed since `0.9.0`: https://github.com/Redocly/openapi-cli/commit/a1b7bd5406c83ca9fea7d029e1d63788de04e59e

**Comment by stevenj** [comment]
The version is 0.12.1.

This is the reference from my main openapi yml file.
```yaml
paths:

  /v1/adapter/claim:
    $ref: "v1/adapter/claim.oas3.yml#/paths/~1adapter~1claim"  
```

The path as defined in v1/adapter/claim.oas3.yml
```yaml
paths:
  /adapter/claim:
    post:
      operationId: adapterClaim
      tags:
        - Adapter Management
```

All of my sub-files are properly constructed openapi3 files, not just snippets.

**Comment by RomanHotsiy** [comment]
Fixed in `0.12.4`

**Comment by stevenj** [comment]
Confirmed 0.12.4 fixes the issue.  That was an amazingly fast fix.  Thank you.
